src/chocolatey/infrastructure.app/tasks/RemovePendingPackagesTask.cs

```diff
@@ -60,13 +60,13 @@ private void handle_message(PreRunMessage message)
         {
             this.Log().Debug(ChocolateyLoggers.Verbose, "[Pending] Removing all pending packages that should not be considered installed...");
 
-            var pendingFiles = _fileSystem.get_files(ApplicationParameters.PackagesLocation, ApplicationParameters.PackagePendingFileName, SearchOption.AllDirectories).ToList();
+            var pendingFiles = _fileSystem.get_files(ApplicationParameters.PackagesLocation, ApplicationParameters.PackagePendingFileName).ToList();

```

Comment:
Is the default to search top level only? It would be better to make it explicit here - `SearchOption.TopDirectoryOnly` (or whatever the correct option is).

Reply:
Yes, SearchOption.TopDirectoryOnly is the default option. It might be better to make it explicit in case that ever changes, but Resharper does mark it as redundant code since it's the default already. Let me know if you want this changed or not.